calculator/main.py

```diff
@@ -26,7 +29,29 @@ def perform_operation(calc, operation, params):
     :param params: Tuple containing the list of nums to operate with.
                    ie: (1, 2, 3, 4.5, -2)
     """
-    pass
+    
+    # Test parameters, raise error if invalid
+    if operation != 'plot':
+        valid_params = all(
+                [isinstance(param, numbers.Number) for param in params]
```

Comment:
That's a good idea!  That's probably a cleaner way of using built-in error handling rather than trying to reinvent it.  Thanks for the note!